.github/workflows/test.yml

```diff
@@ -11,16 +11,30 @@ jobs:
   build:
     runs-on: ubuntu-latest
     env:
-      MEMGRAPH_VERSION: 2.3.1
+      MEMGRAPH_VERSION: 2.4.2
     strategy:
       matrix:
         architecture: ["amd64", "arm64"]
         target: ["prod", "dev"]
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout repository and submodules
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          submodules: recursive
+          token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v1
+      
+      - name: Get Memgraph release version
+        run: |
+          cd cpp/memgraph
+          git fetch --all --tags
+          MEMGRAPH_DB=$(git rev-parse --verify HEAD | git tag --contains)
+          echo $MEMGRAPH_DB
+          echo "MEMGRAPH_DB=${MEMGRAPH_DB:1}" >> $GITHUB_ENV
+
```

Comment:
Will there be a clash since in the step `Download memgraph binaries` we are using Memgraph 2.4.2 and I presume here we are using the latest tag?

Reply:
echo "MEMGRAPH_DB=${MEMGRAPH_DB:1}" >> $GITHUB_ENV

What does this command do?